src/PrestaShopBundle/Service/TranslationService.php

```diff
@@ -35,6 +35,7 @@
 
 class TranslationService
 {
+    const DEFAULT_THEME = 'classic';
```

Comment:
There is a similar constant in `FrontOfficeProvider` you might want to reuse it.